glutin/src/context.rs

```diff
@@ -64,17 +64,26 @@ pub trait NotCurrentGlContext: Sealed {
         surface: &Self::Surface<T>,
     ) -> Result<Self::PossiblyCurrentContext>;
 
-    /// The same as [`Self::make_current`], but provides a way to set read and
-    /// draw surfaces.
+    /// The same as [`Self::make_current`], but provides a way to set draw and
+    /// read surfaces.
     ///
-    /// # Api-specific:
+    /// # Api specific
     ///
     /// - **WGL/CGL:** not supported.
     fn make_current_draw_read<T: SurfaceTypeTrait>(
         self,
         surface_draw: &Self::Surface<T>,
         surface_read: &Self::Surface<T>,
     ) -> Result<Self::PossiblyCurrentContext>;
+
+    /// Make context current on the calling thread without a default
+    /// framebuffer producing the [`Self::PossiblyCurrentContext`].
```

Comment:
Just noting here, but this applies to all comments: I initially read this as "it is being made current on the calling thread which is also_ producing the PossiblyCurrentContext". Rather than that `PossiblyCurrentContext` is the _product of_ such "make current" operation. Perhaps we can reword this and do it across all similar comments, though I'm out of ideas what it should be.

```suggestion
    /// Make context current on the calling thread without a default
    /// framebuffer, producing the [`Self::PossiblyCurrentContext`].
```

Or:


```suggestion
    /// Make context current on the calling thread without a default
    /// framebuffer, and produce the [`Self::PossiblyCurrentContext`].
```